res/database/Default/user/scr0llbaer_news.xml

```diff
@@ -1063,10 +1267,12 @@
 				<band>
 					<de>Band|Gang|Kapelle|Combo|Band</de>
 					<en>Band|Gang|Band|Combo|Band</en>
+					<pl>Band|Gang|Band|Combo|Band</pl>
 				</band>
 				<against>
 					<de>Gegen|Über|Durch|Auf|Für|Aber|Doch,|Doch nicht|Nicht doch|Warum|Wozu|Nicht|Nein,</de>
 					<en>Against|About|Through|To|For|But|Yet,|But not|Oh no, not|Why|What for|Not|No,</en>
+					<pl>Przeciwko|Przez|Dla|Ale|Jeszcze|Ale nie|O nie, nie|Dlaczego|Nie|Nie,</pl>
```

Comment:
this could cause problems, as the number of alternatives differs from the other languages

Reply:
This file I need to review again more carefully